RocketChat/Rocket.Chat.ReactNative
# [FIX] Add search and fix pagination for omnichannels departments

## Proposed changes

When an agent tries to forward an ongoing chat to another department, the list shows only 50 departments. So we are proposing to the agent should be able to either search on the list or see the full list

## Issue(s)	


## How to test or reproduce


- To do a better test for this task, use the mobile.rocket.chat
- Activate as an online agent
- To create a new chat you can go to https://mobile.rocket.chat/livechat
- Then, take the chat from queue and go to actions room and click to forward the chat

- You should be able to see the pagination for departments
- Add to this, you can search by typing the department's name

## Screenshots

### Before

https://user-images.githubusercontent.com/47038980/150029564-712332b1-e426-481b-aa08-789e824b4c78.mov

### After

https://user-images.githubusercontent.com/47038980/150029612-d5920d82-95db-4250-9115-f56bca8ea037.mov


## Types of changes




## Checklist



## Further comments
